plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/plugins/index.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { SelectControl } from '@wordpress/components';
+import {
+	// @ts-expect-error no exported member.
+	PluginDocumentSettingPanel,
+} from '@wordpress/editor';
+/**
+ * Internal dependencies
+ */
+import useProductTypeSelector from '../hooks/use-product-type-selector';
+
+export const productTypeSelectorTitle = __( 'Product Type', 'woocommerce' );
```

Comment:
Do we need to declare and export this `const`? I haven't seen it used anywhere else.

Reply:
Yes, I think we can remove the export for now until we have a need to export it.